R/calculate_silhouette_width.R

```diff
@@ -0,0 +1,77 @@
+#' Calculate silhouette width scores from an integrated SCE object.
+#' This function performs replicated calculations on downsampled data.
```

Comment:
It looks like the documentation function isn't working properly (and the test failure is for documentation mismatches): Can you rerun documentation on this @cbethell?

I'm also separating the "title" from "description" here by adding a blank line in the documentation script.

```suggestion
#'
#' This function performs replicated calculations on downsampled data.
```